scripts/promote-tracks.py

```diff
@@ -0,0 +1,84 @@
+import requests
```

Comment:
This is missing, and is the reason for the cryptic error message you see on the CI.

```suggestion
#!/usr/bin/env python3

import requests
```